### ISSUE TITLE
Confirm DB backup strategy

### ISSUE THREAD

**Issue by rusefi** [rusefi/web_backend#72]
REO lives in the same DB with forum and legacy wiki. 
At the moment I only manually export DB at random points in time

it looks like web hosting keeps 6 days of daily FILE backups but not sure about DB

**Comment by rusefi** [comment]
Between https://www.ionos.com/help/hosting/backup-restore-files/restoring-deletedoverwritten-files-for-linux-web-hosting-packages/ and https://www.ionos.com/help/hosting/using-mysql-databases-for-web-projects/backing-up-mysql-databases-with-phpmyadmin/  I am starting to think that 1and1 by ionos provides zero, non, nada automated database backups

First question is where do we want to backup TO.
Second question is automation script probably - you've guessed it - as GHA action and/or Jentkins?

**Comment by rusefi** [comment]
@ksmola @mck1117 @andreika-git @chuckwagoncomputing ideas on ^^^?

**Comment by chuckwagoncomputing** [comment]
I don't think this is a job for CI scripts. I assume you have access to the DB from your SSH account? If so, you should be able to write a script that is run by a cron job, that backs up the DB and uploads it to somewhere. I'm guessing you don't have the ability to install software, and will have to make do with what you have.

There are a lot of advantages to running your own server. Personally, I rent a couple of servers from Linode, and have been very happy with them. The disadvantage in our case is time spent administrating them which could have been spent improving the code.
